uspace/lib/c/include/fibril_synch.h

```diff
@@ -142,6 +142,29 @@ typedef struct {
 	void *arg;
 } fibril_timer_t;
 
+/**
+ * A counting semaphore for fibrils.
+ * https://en.wikipedia.org/wiki/Semaphore_(programming)
```

Comment:
I would drop the wikipedia reference. Everyone knows what a semaphore is.

Reply:
Okay.